.travis.yml

```diff
@@ -20,9 +21,12 @@ script:
 matrix:
   include:
   # Include sync acceptance tests for node 4
-  - node_js: "4.4.3"
+  - node_js:
+      - 4
```

Comment:
same as above re node4

Reply:
@damienomurchu 👍 tks we need so re-update all really tks to check it.